send.go

```diff
@@ -376,6 +413,23 @@ func DecodeABI(v []byte, args ...interface{}) error {
 				copy(s[i][:], cur[o+12:o+32])
 			}
 			*v = AddrSlice(s)
+		case *DataSlice:
```

Comment:
You should also add a case for `*Bytes` in this switch.

Reply:
Done.